__init__.py

```diff
@@ -184,19 +184,20 @@ def rate_on_ankiweb(did=None):
 
 
 def visualize_schedule(did=None):
+    url = "https://open-spaced-repetition.github.io/anki_fsrs_visualizer"
     deck = mw.col.decks.current()
-    config = mw.col.decks.get_config(deck["conf"])
-    retention = config["desiredRetention"]
-    fsrs_params = (
-        config["fsrsParams5"]
-        if "fsrsParams5" in config and len(config["fsrsParams5"]) > 0
-        else config["fsrsWeights"]
-    )
-    fsrs_params_string = ",".join(f"{x:.4f}" for x in fsrs_params)
-
-    openLink(
-        f"https://open-spaced-repetition.github.io/anki_fsrs_visualizer/?w={fsrs_params_string}&m={retention}"
-    )
+    if "config" in deck:
+        config = mw.col.decks.get_config(deck["conf"])
+        retention = config["desiredRetention"]
+        fsrs_params = (
+            config["fsrsParams5"]
+            if "fsrsParams5" in config and len(config["fsrsParams5"]) > 0
+            else config["fsrsWeights"]
```

Comment:
`len(config["fsrsWeights"])` could also be 0.

Reply:
What's your suggestion for that case? Fallback to the default params?

Reply:
No need to handle it; `ts-fsrs` will use default parameters.

https://github.com/open-spaced-repetition/ts-fsrs/blob/af8f8732f62217c981bee8fa52ace4ed58cea67b/src/fsrs/default.ts#L19-L34